nautobot/core/api/metadata.py

```diff
@@ -72,41 +77,47 @@ def _get_field_properties(self, field: schema.SerializerType, name: str) -> Dict
             result["items"] = self._get_field_properties(field.child_relation, "")
             result["uniqueItems"] = True
         else:
+            if isinstance(field, drf_serializers.RelatedField):
+                result["uniqueItems"] = True
+                if hasattr(field, "queryset") and hasattr(field.queryset, "model"):
+                    model_options = field.queryset.model._meta
+                    result["required"] = field.required
+                    # Custom Keyword: modelName and appLabel
+                    # This Keyword represents the model name of the uuid model
+                    # and appLabel represents the app_name of the model
+                    result["modelName"] = model_options.model_name
+                    result["appLabel"] = model_options.app_label
```

Comment:
Do we need to add these keys for the `ManyRelatedField` case above as well?

Reply:
Nope, as ManyRelated field, would is a recursive call, and would end up getting to this point